homeworks/homework_2/src/components/Video.js

```diff
@@ -0,0 +1,18 @@
+import React, {Component} from 'react';
+
+export class Video extends Component {
+    constructor (props) {
+        super(props)
+    }
+    render () {
+        return(
+            <React.Fragment>
```

Comment:
I guess you could check if videoId passed there, you show component or render nothing